examples/hybrid/bubble_2d_fieldbc.jl

```diff
@@ -0,0 +1,321 @@
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
```

Comment:
if this file isn't going to be part of a test, it should be deleted

Reply:
What do you think is appropriate? It does demonstrate the two ways to set BCs.

Reply:
It would be better to have a unit test if possible.